src/server/pfs/server/val_server.go

```diff
@@ -36,20 +36,17 @@ func (a *validatedAPIServer) DeleteRepoInTransaction(txnCtx *txncontext.Transact
 }
 
 // FinishCommitInTransaction is identical to FinishCommit except that it can run
-// inside an existing etcd STM transaction.  This is not an RPC.
+// inside an existing postgres transaction.  This is not an RPC.
```

Comment:
It’s really a Pachyderm transaction, isn’t it? Because the commit could have been started in an entirely different RPC originally, right?